ignite/cmd/network_request_approve.go

```diff
@@ -21,8 +21,23 @@ func NewNetworkRequestApprove() *cobra.Command {
 		Use:     "approve [launch-id] [number<,...>]",
 		Aliases: []string{"accept"},
 		Short:   "Approve requests",
-		RunE:    networkRequestApproveHandler,
-		Args:    cobra.ExactArgs(2),
+		Long: `The "approve" command is used by a chain's coordinator to approve requests.
+Multiple requests can be approved using a comma-separated list and/or using a
+dash syntax.
+
+  ignite network request approve 42 1,2,3-6,7,8
+
+The command above approves requests with IDs from 1 to 8 included on a chain
+with a launch ID 42.
+
+When requests are approved Ignite applies the requested changes and simulates
```

Comment:
```suggestion
When requests, are approved Ignite applies the requested changes and simulates
```

Reply:
I don't think a comma should be there tbh

Reply:
How about 

"When requests are removed, Ignite ...."

?

Reply:
Yeah my bad I rather wanted to suggest what Alex mention
But not a big deal